### PR TITLE
Add humanize 0.5.1

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
@jmoiron I am adding humanize to conda-forge for linux, mac and windows. This will allow users to easily install this with [conda] (http://conda.pydata.org/docs/) (conda install -c conda-forge humanize). The maintenance burden here is light as all testing and releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/humanize-feedstock shortly after this PR is finished and merged. If you would like to be added as a maintainer (to make new releases for instance), I can happily add you to the maintainers list. If not, that is totally fine too!

File: recipes/humanize/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  name: humanize
+  version: {{ version }}
+
+source:
+  fn: humanize-{{ version }}.tar.gz
+  url: https://github.com/jmoiron/humanize/archive/{{ version }}.tar.gz
+  sha256: e3f3ac75e647f75eec48c3950385ab7585c1c1cde2a6c1479c1f58e055a3e868
+
+build:
+  number: 1
```
Comment (jjhelmus) [inline recipes/humanize/meta.yaml]: Typically we start with build number 0

Comment (ericdill) [inline recipes/humanize/meta.yaml]: Good catch, thanks @jjhelmus

**Comment by jjhelmus** [comment]
LGTM

**Pull request merged**